locale/ko.js

```diff
@@ -49,8 +49,22 @@ var ko = moment.defineLocale('ko', {
         y : '일 년',
         yy : '%d년'
     },
-    ordinalParse : /\d{1,2}일/,
```

Comment:
Please rebase from the HEAD of the develop branch. This was renamed `dayOfMonthOrdinalParse` 5 days ago.